bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/connection/AbstractJeeLinkConnection.java

```diff
@@ -94,7 +89,11 @@ public void sendCommands(String commands) {
                 // in case of tcp connections, the underlying socket
                 OutputStreamWriter w = new OutputStreamWriter(initStream);
                 for (String cmd : initCommands) {
-                    w.write(cmd);
+                    if (logger.isDebugEnabled()) {
```

Comment:
You can remove that check. It was there because `Arrays.toString()` is expensive. `port` and `cmd` are not, because `toString()` is called only if the log-level is debug or higher.

Reply:
Okay

Reply:
Okay